tests/View/Blade/BladeRouteIsTest.php

```diff
@@ -0,0 +1,13 @@
+<?php
+
+namespace Illuminate\Tests\View\Blade;
+
+class BladeRouteIsTest extends AbstractBladeTestCase
+{
+    public function testRouteIsCompiled()
+    {
+        $string = '@routeIs(foo) bar @endRouteIs';
+        $expected = '<?php if(request()->routeIs(foo)): ?> bar <?php endif; ?>';
```

Comment:
```suggestion
        $string = '@routeIs('foo', 'bar') baz @endRouteIs';
        $expected = '<?php if (request()->routeIs('foo', 'bar')): ?> baz <?php endif; ?>';
```

Adjust the test for valid PHP syntax since `@routeIs(foo)` would throw an exception for an undefined PHP constant.